test/integration/test_fake_backends.py

```diff
@@ -183,8 +183,25 @@ def setUpClass(cls):
 
     def test_refresh_method(self):
         """Test refresh method"""
+        # to verify the data files will be updated
         old_backend = FakeSherbrooke()
-        with self.assertLogs("qiskit_ibm_runtime", level="INFO"):
+        # change some configuration
+        old_backend.backend_version = "fake_version"
+        with self.assertLogs("qiskit_ibm_runtime", level="INFO") as logs:
             old_backend.refresh(self.service)
+        self.assertIn("The backend fake_sherbrooke has been updated", logs.output[0])
+
+        # to verify the data files are currently updated that there is nothing to refresh
+        # create another instance of FakeSherbrooke updated above
         new_backend = FakeSherbrooke()
-        self.assertGreaterEqual(old_backend.backend_version, new_backend.backend_version)
+        with self.assertLogs("qiskit_ibm_runtime", level="INFO") as logs:
+            new_backend.refresh(self.service)
+        self.assertIn("There are no available new updates for fake_sherbrooke", logs.output[0])
+
+        # to verify the refresh can't be done
+        wrong_backend = FakeSherbrooke()
+        # set a non-existent backend name
+        wrong_backend.backend_name = "wrong_fake_sherbrooke"
+        with self.assertLogs("qiskit_ibm_runtime", level="WARNING") as logs:
+            wrong_backend.refresh(self.service)
+        self.assertIn("The refreshing of wrong_fake_sherbrooke has failed", logs.output[0])
```

Comment:
Tested locally and functionally LGTM but is there a way to revert the files that are updated when running this test? Not sure how this works in the github CI but when running the tests locally it'd be nice if they didn't edit local files